understanding/21/target-size-enhanced.html

```diff
@@ -19,6 +19,7 @@ <h2>Intent</h2>
 			  <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
 			  <li>the control is part of a sequential task.</li>
 	  </ul>
+	  <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size as the target. It is recommended that when the visible indicator of the target is notably smaller than the target, that the visible indicator be placed at the center of the target.</p>
```

Comment:
As per my comments in the issue, I'd prefer we not get into this space. But if we're going to add some content, I don't see why it is only germane to small indicators. We'd want big indicators to be centered on the target too.
```suggestion
	  <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size or shape as the target. It is recommended that when the visible indicator of the target differs from the target, that the visible indicator be centered on the target.</p>
```